enterprise/server/self-hosting-hunspell.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Add Hunspell dictionaries to Spell Checker Pro
+title_nav: Using Hunspell dictionaries
+description: Instructions for adding Hunspell dictionaries to TinyMCE Spell Checker Pro.
+keywords: enterprise tinymcespellchecker hunspell spell check checker pro server configuration configure
+---
+
+{{site.requires_5_9v}}
+
+{{site.productname}} 5.9 introduced support for Hunspell dictionaries. Hunspell dictionaries can be obtained from various sources, but the files must be stored in a specific structure to work with {{site.productname}} Spell Checker Pro. {{site.companyname}} provides downloadable bundles that have the required structure.
+
+To add Hunspell dictionaries to a self-hosted {{site.productname}}:
+
+1. Download the latest {{site.productname}} Enterprise package.
+1. Download the desired Hunspell dictionaries.
+1. Configure the {{site.productname}} spelling service to use the Hunspell dictionaries.
+
+## Downloadable Hunspell Bundles
+
+{{site.companyname}} provides two downloadable bundles of Hunspell dictionaries.
+
+`hunspell-dictionaries-approved.zip`
+: Does **not** contain dictionaries licensed under:
+* [The GNU General Public License (**GPL**)](https://www.gnu.org/licenses/licenses.html#GPL).
+* [The GNU Lesser General Public License (**LGPL**)](https://www.gnu.org/licenses/licenses.html#LGPL).
+
+`hunspell-dictionaries-all.zip`
+: This package contains all the Hunspell dictionaries that the spelling service supports. You will need to ensure that their license matches your requirements.
+
+Hunspell dictionaries can be downloaded from other sources, but will need to be stored in the structure shown in [Hunspell dictionary storage for Spell Checker Pro](#hunspelldictionarystorageforspellcheckerpro).
```

Comment:
Mentioning this feels like it's only going to lead to issues. Are we sure we want to do this and if we do, do we need to put a disclaimer that not all languages would work? (e.g. any CJK variant won't work due to the client word detection logic)

Reply:
Either that, or maybe we need to re-iterate the supported languages part, as this is detached and reading it, it comes off as you can use any dictionary and it'll work.

Reply:
```suggestion
Hunspell dictionaries can be downloaded from other sources, but will need to be stored in the structure shown in [Hunspell dictionary storage for Spell Checker Pro](#hunspelldictionarystorageforspellcheckerpro). Not all Hunspell dictionary languages work with Spell Checker Pro, for a list of supported languages, see: [Spell Checker Pro plugin - Supported languages]({{site.baseurl}}/plugins/premium/tinymcespellchecker/#supportedlanguages).
```